Documentation/Troubleshooting/krew-plugin.md

```diff
@@ -1,69 +0,0 @@
----
```

Comment:
Let's actually keep this doc for now. It's ok for visibility to help them install it, I just don't want to duplicate more commands to this page, so they can refer to the krew docs for the commands.

Reply:
reverted.